ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/DescribeFunctionExecutor.java

```diff
@@ -85,6 +91,34 @@ private static FunctionDescriptionList describeAggregateFunction(
     );
   }
 
+  private static FunctionDescriptionList describeTableFunction(
+      final KsqlExecutionContext executionContext,
+      final String functionName,
+      final String statementText
+  ) {
+    final TableFunctionFactory tableFunctionFactory = executionContext.getMetaStore()
+        .getTableFunctionFactory(functionName);
+
+    final ImmutableList.Builder<FunctionInfo> listBuilder = ImmutableList.builder();
+
+    tableFunctionFactory.eachFunction(func -> listBuilder.add(
```

Comment:
this code is essentially duplicated from below - can we extract what we need into an interface that both `TableFunctionFactory` and `UdfFactory` implement and then just delegate to a helper method?

Reply:
I can improve it a bit, but FunctionInfo lives in the rest module and is not accessible from TableFunctionFactory or AggregateFunctionFactory

Reply:
fixed